src/writer/serve.py

```
@@ -363,95 +395,217 @@ async def _get_payload_as_json(request: Request):
             raise HTTPException(status_code=400, detail="Cannot parse the payload.")
         return payload
 
-    @app.post("/api/job/blueprint/{blueprint_key}")
+    @app.post("/private/api/job/blueprint/{blueprint_key}")
     async def create_blueprint_job(blueprint_key: str, request: Request, response: Response):
```

Pylint (3.3.7)

[refactor] Too many statements (58/50)

(R0915)